python/cudf/groupby/legacy_groupby.py

```diff
@@ -562,6 +562,74 @@ def mult_add(key, val, out1, out2):
             5    2    5   10    7
             6    2    6   12    8
 
+
+
+        .. code-block:: python
+
+            from cudf import DataFrame
```

Comment:
To avoid reader confusion between cudf.DataFrame and pandas.DataFrame, can you please change this to `import cudf`?